bumble/transport/common.py

```diff
@@ -132,7 +132,9 @@ def feed_data(self, data: bytes) -> None:
                         packet_type
                     ) or self.extended_packet_info.get(packet_type)
                     if self.packet_info is None:
-                        raise ValueError(f'invalid packet type {packet_type}')
+                        logger.warning(f'ignoring invalid packet type: {packet_type}')
```

Comment:
the tradeoff here is that if there's a data source with corrupted data, the parser will go on forever without ever throwing an exception. I think that's Ok. If we encounter cases where that happens, we can always enhance the parser with a property that configures how much  invalid data it is willing to ignore.

Reply:
I think I found that when I got invalid packet type data in back-to-back packets, eventually I got a valid packet, which allowed me to continue. With the exception, the background thread crashed and my application code wasn't aware of it without an `wait asyncio.wait_for()` wrap-around the `.power_on()` method. I think it's possible with this change that the rest of the packet data could contain a start of a valid packet after the invalid data, and since we don't throw the exception, now we miss it, and end up in the same situation of waiting indefinitely, but I think in that case, the behaviour is the same.